react/src/components/Home.js

```diff
@@ -1,20 +1,18 @@
 import { Component } from 'react';
```

Comment:
'Component' is defined but never used. I think this can be deleted.

Reply:
addressed in https://github.com/sentry-demos/empower/pull/368/commits/ff645dc059c1f9966870698159a6dd4ea99c9d3d